galley/docker/Dockerfile.galley

```diff
@@ -1,3 +1,13 @@
-FROM istionightly/base_debug
+# BASE_DISTRIBUTION is used to switch between the old base distribution and distroless base images
+ARG BASE_DISTRIBUTION=default
+
+# -----------------------------------------------------------------------------
+FROM istionightly/base_debug as default
```

Comment:
Does adding these FROMS make our image bigger (I think definitely no) or make our build times slower? Or is it effectively a NOP for the one we don't use.

If it does have an impact, maybe we can do `FROM $BASE` and have the arg be passed in as the full image?

Reply:
Looking more I see the issue could be the default is not always the same

Reply:
It does not increase the image size. But make becomes slower. When both versions are build and pushed it is significantly slower. Nevertheless it is only an interim solution.

Reply:
What is the long term solution?

Reply:
The long term solution would be to remove the `default` section and only use the `distroless` version.

Reply:
Am I understanding you correctly that you are saying, in the long term, we will only offer a distroless base and not a debian base?

I have not heard this before

Reply:
In my point of view, that would be the right direction. This was discussed under #12122.

Reply:
I don't see any discussion of default in that issue. In all the WG meetings we have discussed how we want more options, not a different option, so I am not sure where you got the idea it would be the default from. The tools in the Debian image we provide are essential to debugging issues users face, and we use them daily.

I think everyone is really excited to have the option to use distroless, but not if it comes at the expense of te Debian one (which it doesn't have to, we can build any number of images)

Reply:
Sorry I was not part of the discussion. Perhaps, it was only my impression, that the direction should be a `distroless` based set of images.
I'm also fine with having both options in place.

Reply:
I thought you were, hard to match a meeting voice to a github username sometimes though.

Thanks for pushing through all the pushback on the PR, this will be a great addition to Istio

Reply:
Only the standard images will be built with default configuration.